apps/web/src/routes/_content/resources/podcasts/$podcastName-$podcastId.tsx

```diff
@@ -115,7 +116,12 @@ function Podcast() {
           />
 
           <article className="w-full">
-            <Card className="md:mx-auto" color="orange">
+            <Card
+              className="md:mx-auto w-full max-w-[1179px]"
+              withPadding={false}
+              paddingClass="p-5 md:p-[30px]"
+              color="orange"
+            >
               <div className="w-full flex flex-col md:flex-row gap-5 lg:gap-16">
                 <div className="flex flex-col items-center justify-center">
                   <img
```

Comment:
Image on desktop shouldn't be rounded according to Figma but it may be a mistake as it is rounded on mobile design, could you quickly check what the expected behavior is ?

Reply:
Margin bottom under image is too big, should be mb-[30px] on desktop and mb-5 on mobile
![image](https://github.com/user-attachments/assets/95378204-8499-4166-a0fd-28428179bae6)
![image](https://github.com/user-attachments/assets/c1f8d003-1d17-4486-887e-df19ed5c4648)

Figma -> 
![image](https://github.com/user-attachments/assets/d31e10fc-c5f8-48d4-b439-a4302c15fe50)
![image](https://github.com/user-attachments/assets/9a7a00b6-cfd9-4f44-a00b-ee14ef80c946)